monkey/monkey_island/cc/services/agent_plugin_service/flask_resources/agent_plugin_installation.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from http import HTTPStatus
+
+from flask import make_response, request
+from flask_security import auth_token_required, roles_accepted
+
+from monkey_island.cc.flask_utils import AbstractResource
+from monkey_island.cc.services.authentication_service import AccountRole
+
+from .. import IAgentPluginService
+from ..errors import PluginInstallError
+
+logger = logging.getLogger(__name__)
+
+
+class AgentPluginInstallation(AbstractResource):
```

Comment:
I'm not a fan of this. This is neither RESTful nor RPC-style. Why aren't we just implementing PUT in the `AgentPlugins` resource that we already have?

Reply:
If there's a good reason for not doing that, we should change this to RPC-style (`InstallAgentPlugin`).